event_registration_partner_unique/models/event_registration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2016 Antiun Ingeniería S.L. - Jairo Llopis
```

Comment:
Don't split this file. This can be considered a one2many of `event.event`, and thus, it can go together

Reply:
`event_regisration.py` file nested inside `event.py` file